packages/vue3-vite/src/generators/library/files/tsconfig.spec.json.template

```diff
@@ -6,5 +6,5 @@
     "types": ["vue-i18n", "jest", "node"],
     "composite": true
   },
-  "include": ["**/*.spec.ts", "**/*.spec.tsx", "**/*.d.ts"]
+  "include": ["**/*.spec.ts", "**/*.spec.tsx", "**/*.d.ts", "**/*.ts"]
```

Comment:
I have to add `**/*.ts` here to make the app "sees" the `index.ts` in lib projects.

Reply:
That's odd, I wonder why it's not necessary in the libraries generated by `@nrwl/node:lib`.